openlibrary/plugins/upstream/tests/test_merge_authors.py

```diff
@@ -1,16 +1,13 @@
-import web
-
 from infogami.infobase import client, common
-from infogami.utils import delegate
+from openlibrary.plugins.upstream.merge_authors import *
```

Comment:
PEP8 says:
> Wildcard imports (`from <module> import *`) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn't known in advance).

flake8 will rightly complain about them because they disable all _undefined name- checks.  For instance, `import web` has been removed yet `web` is still referred to on lines 68, 144, 158, 160, 234, 237, 253 but flake8 can not tell if `web` is part of the wildcard import or is a standalone module.

https://www.asmeurer.com/removestar is a tool that can automated the process.

Reply:
Fixed (semi-manually using PyCharm to import when it found an error)